### PR TITLE
Remove symbolsource no longer relevant

### PR DESCRIPTION
**Pull request by erdembayar**
Fixes: https://github.com/NuGet/Client.Engineering/issues/1692

Removing references to symbolsource, follow up on https://github.com/NuGet/docs.microsoft.com-nuget/pull/2855.

### CODE REVIEW

File: docs/reference/cli-reference/cli-ref-push.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ where `<packagePath>` identifies the package to push to the server.
 
 - **`-SymbolSource`**
 
-  *(3.5+)* Specifies the symbol server URL; nuget.smbsrc.net is used when pushing to nuget.org
+  Specifies the symbol server URL.
```
Comment (erdembayar) [inline docs/reference/cli-reference/cli-ref-push.md:73]: Make it same as `dotnet nuget push` text.

File: docs/release-notes/NuGet-1.3.md [hunk 2]
```diff
@@ -15,16 +15,6 @@ NuGet 1.3 was released on April 25, 2011.
 
 ## New Features
 
-### Streamlined Package Creation with symbol server integration
```
Comment (erdembayar) [inline docs/release-notes/NuGet-1.3.md:18]: This service no longer works.

Comment (jeffkl) [reply docs/release-notes/NuGet-1.3.md:18]: Team review: add a note that this service is no longer supported

Comment (erdembayar) [reply docs/release-notes/NuGet-1.3.md]: I added note : `The above is deprecated and no longer supported.`
Please review again.

**Review by nkolev92 — APPROVED** [review]

**Pull request merged**